data/inline-histogram_test.go

```diff
@@ -168,3 +168,31 @@ func TestAddObservationNormalDistConvergenceMostStrict(t *testing.T) {
 	fmt.Println(fmt.Sprintf("variance %f", ih.pm.GetSampleVariance()))
 	fmt.Println(fmt.Sprintf("sample size %d", ih.pm.GetSampleSize()))
 }
+
+func TestBins(t *testing.T) {
+	ih := Init(1.0, 0.0, 5.0)
```

Comment:
bin width of 1 with a min value of zero should pass the test... try a min value of not zero and a width of not 1.

Reply:
Done. I'll push after I fix the above issues.